### ISSUE TITLE
Remove date from dataset name

### ISSUE THREAD

**Issue by blinkseb** [cp3-llbb/GridIn#33]
We should remove the date from the sample name and append it automatically from the submit script if it's needed.

**Comment by OlivierBondu** [comment]
I tend to disagree with this one... I prefer to include the date of the release of the json file instead of the date of the submission of the jobs ?
